docs/infrastructure/hosted-server.rst

```diff
@@ -84,12 +84,30 @@ When creating a new VM from a template…
 #. Start VM, follow any setup scripts
 #. Profit!
 
+
+**************
 Use and Access
-==============
+**************
 
-Use and access is limited to members of the executive board.
+Use and access of Titan is limited to the President, Vice President, and Project Coordinator.
 In the future, there may be opportunities for club members to participate in managing RITlug's infrastructure.
 
+How to Gain Access to Titan
+===========================
+
+Access to Titan first requires access to the VPN, as Titan's Management Interface (Proxmox) is IP restricted.
+To get on the VPN, Solomon needs to give you the OpenVPN file to initiate the connection.
+This VPN does not allow anything but local traffic through, so tunneling does not work.
+
+Lastly, Proxmox itself needs an account.
+Proxmox is used in order to avoid unnecessary unix accounts.
+This requires a username, and Solomon provides a temporary password that should be changed upon log in.
+So, for approved users, this is the process for gaining access to Titan:
```

Comment:
This is phrased awkwardly in my opinion. I would drop the "so" and "this is" and add something like "is as follows" to the end. Not going to block on this though and there are probably better ways to word it than I'm suggesting too.

Reply:
Hmm, yes this is phrased slightly awkwardly. However, I don't think it is a blocker to merge this.